### ISSUE TITLE
support for Mac M1

### ISSUE THREAD

**Issue by wind57** [vmware-tanzu/homebrew-kpack-cli#1]
is there a plan to support Mac M1 by any chance? Currently:

```
wind57@wind57s-MacBook-Pro ~ [1]> brew install vmware-tanzu/kpack-cli/kp
kp: The x86_64 architecture is required for this software.
Error: kp: An unsatisfied requirement failed this build.
```

thank you

**Comment by jferna57** [comment]
I have the same problem. Would be great!

**Comment by chenbh** [comment]
arm64 support for both linux and osx has been available since 134562da4195ed9f24e5dfe3effa183dcefacc0a (v0.9.1)
